connect_transformations/currency_conversion/mixins.py

```diff
@@ -13,6 +13,36 @@
 
 class CurrencyConverterTransformationMixin:
 
+    async def _get_current_exchange_rate(self, currency_from, currency_to):
+        rate = getattr(self, 'current_exchange_rate', None)
+        if rate:
+            return rate
+        try:
+            params = {
+                'from': currency_from,
+                'to': currency_to,
+                'amount': 1,
+            }
+            async with httpx.AsyncClient(
+                transport=httpx.AsyncHTTPTransport(retries=3),
+            ) as client:
+                response = await client.get(
+                    'https://api.exchangerate.host/convert',
```

Comment:
Probably it's better to consume the following API to get the conversion rate:
`https://api.exchangerate.host/latest?base=EUR&symbols=USD`

Documentation here: https://exchangerate.host/#/#our-services